src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -333,6 +362,10 @@ public static boolean allowsHookUrlOverride() {
             return ALLOW_HOOKURL_OVERRIDE;
         }
 
+        private static ThreadFactory threadFactory() {
```

Comment:
Don't think the complexity reduction in such simple way is "like change for change's sake". At minimal state it should be named as `namedThreadFactory` as of its simple shortcut.